src/main/java/org/solid/testharness/reporting/Assertion.java

```diff
@@ -29,7 +29,7 @@ public String getMode() {
 
     public TestResult getResult() {
         final List<TestResult> results = getModelList(EARL.result, TestResult.class);
-        if (!results.isEmpty()) {
+        if (results != null) {
```

Comment:
I don't know what the difference might be

Reply:
isEmpty assumes the list was instantiated and I realised it was more accurate to say it was null, not existing but empty.
null means the list was not instantiated and calling any method would cause an exception.